### PR TITLE
Adapt to BlueBrain/RESTBridge#40 (new RestBridge API)

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
Do not merge, just review

**Comment by eile** [comment]
@tribal-tec FYI

File: livre/eq/zeq/communicator.cpp [hunk 1]
```diff
@@ -51,7 +48,7 @@ namespace zeq
 class Communicator::Impl
 {
 public:
-    explicit Impl( Config& config )
+    explicit Impl( Config& config, const int argc, char** argv )
```
Comment (tribal-tec) [inline livre/eq/zeq/communicator.cpp]: not anymore explicit

File: livre/eq/zeq/communicator.h [hunk 2]
```diff
@@ -34,7 +34,7 @@ namespace zeq
 class Communicator
 {
 public:
-    explicit Communicator( Config& config );
+    explicit Communicator( Config& config, int argc, char** argv );
```
Comment (tribal-tec) [inline livre/eq/zeq/communicator.h]: dito

Comment (eile) [inline livre/eq/zeq/communicator.h]: done

File: livre/eq/zeq/communicator.cpp (hunk 1 above)
Comment (eile) [inline livre/eq/zeq/communicator.cpp]: done

**Comment by eile** [comment]
pushed into summit branch for now. Will reopen later.

**Pull request closed without merging**